packages/react-utilities/src/utils/getNativeElementProps.ts

```diff
@@ -65,3 +65,31 @@ export function getNativeElementProps<TAttributes extends React.HTMLAttributes<a
 
   return getNativeProps(props, allowedPropNames, excludedPropNames);
 }
+
+/**
+ * Splits the native props into ones that go to the 'root' slot, and ones that go to the 'primary' slot.
+ *
+ * This function is only for use with components that have a primary slot other than 'root'.
+ * Most components should use {@link getNativeElementProps} for their root slot if it is the primary slot.
+ *
+ * @returns An object containing the native props for the `root` and `primary` slots.
```

Comment:
nit, to reflect that `root` is the actual name of a slot in code and "primary" is not
```suggestion
 * Splits the native props into ones that go to the `root` slot, and ones that go to the "primary" slot.
 *
 * This function is only for use with components that have a primary slot other than `root`.
 * Most components should use {@link getNativeElementProps} for their root slot if it is the primary slot.
 *
 * @returns An object containing the native props for the `root` and primary slots.
```

Reply:
Update to have backticks around `root`, and no quotes around primary since it's just an adjective and doesn't need scare quotes.